source/streams/StreamRecording.cpp

```diff
@@ -251,11 +222,3 @@ bool StreamRecording::isStopped()
     fiber_sleep(0);
     return this->state == REC_STATE_STOPPED;
 }
-
-float StreamRecording::getSampleRate()
-{
-    if( this->lastUpstreamRate == DATASTREAM_SAMPLE_RATE_UNKNOWN )
-        return this->upStream.getSampleRate();
-    
-    return this->lastUpstreamRate;
-}
```

Comment:
The `StreamRecording::getSampleRate()` method has been removed by the `virtual float getSampleRate()` declaration has been left in `StreamRecording.h`, which causes a compilation error.